src/Silex/Provider/SwiftmailerServiceProvider.php

```diff
@@ -91,26 +91,30 @@ public function register(Container $app)
 
             $plugins = $app['swiftmailer.plugins'];
 
+            $keys = $plugins->keys();
+            sort($keys);
```

Comment:
@c960657 I am using Silex 2.0.x and I've added an array of swiftmailer plugins to Pimple and I'm registering them lazily. The order I've added them is important for me, though. I have a plugin which for specific emails would add a bcc and another plugin which would blacklist/whitelist certain emails and domains based on environment. So if I register the filtering plugin first and it filters out all emails, the plugin adding bcc would not work, because the filtering plugin will cancel the email event. If I add the bcc plugin first, and it adds a whitelisted email, the filtering plugin will remove all emails and just send an email to the bcc email without canceling the whole email. I don't see how sorting implicitly works towards more predictability.

Other than that, thank you for work on improving the plugins configuration, it is much appreciated!